src/application-manager.d.ts

```diff
@@ -97,7 +97,11 @@ class ApplicationManager extends EventEmitter {
 		transaction: Knex.Transaction,
 	): Promise<void>;
 
-	public getStatus(): Promise<DeviceStatus>;
+	public getStatus(): Promise<{
+		local: DeviceStatus.local.apps;
```

Comment:
Updated typescript caught that we were using the types expecting them to look like this and it turns out that we were using them correctly but typing incorrectly, so I've now fixed the typing